docs/reference/ml/ml-shared.asciidoc

```diff
@@ -1422,6 +1423,131 @@ The number of `partition` field values that were analyzed by the models. This
 value is cumulative for all detectors in the job.
 end::total-partition-field-count[]
 
+tag::trained-model-configs[]
+An array of trained model resources, which are sorted by the `model_id` value in 
+ascending order.
+
+`model_id`:::
+(string)
+Idetifier for the trained model.
+
+`created_by`:::
+(string)
+Information on the creator of the trained model.
+
+`version`:::
+(string)
+The {es} version number in which the trained model was created.
+
+`create_time`:::
+(<<time-units,time units>>)
+The time when the trained model was created.
+
+`tags`:::
+(string)
+A comma delimited string of tags. A {infer} model can have many tags, or none.
+
+`metadata`:::
+(object)
+An object containing metadata about the trained model. Includes an 
+`analysis_config` and an `input` object.
+
+`analytics_config`:::
```

Comment:
none of these `analytics*` entries should be documented. These happen to be returned in models created by an analytics process but are not guaranteed to be present.